include/http/router.js

```diff
@@ -2,15 +2,16 @@ const Koa = require('koa');
 const Router = require('koa-router');
 const Session = require('../koa/Session')();
 const bodyParser = require('koa-body');
-const Cookies  = require('koa-cookie').default;
+const Cookies = require('koa-cookie').default;
 const Passport = require('../koa/authentication/Passport')();
+const https = require('https');
+const fs = require('fs');
 
-
-module.exports = function (pb) {
+module.exports = function(pb) {
 
     class PencilblueRouter {
 
-        constructor () {
+        constructor() {
```

Comment:
minor: check this rule running ESlint as I remember there should be a blank space after the function name and the parenthesis

Reply:
That always gets me.  Especially since WebStorm auto formats to not having that.  With that said, does Core PB have a lint file?

Reply:
Nope, it doesn't have any lint file